frontend/cypress/support/commands.ts

```diff
@@ -4,6 +4,7 @@ import { pressUntilContains, doUntilSelector } from './helper/utils';
 import Chainable = Cypress.Chainable;
 
 Cypress.Commands.add('loginAsUser', (user: any) => {
+  cy.viewport(1920, 1080);
```

Comment:
Wird auf dem `main` bereits in der Cypress-Config gesetzt, auch ein Fehler durch den Rebase?

Reply:
ist auch ein Rebase Fehler